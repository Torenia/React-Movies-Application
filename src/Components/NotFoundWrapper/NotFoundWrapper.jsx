import styled from 'styled-components';

export const NotFoundWrapper = styled.div`
  width: 100%;
  height: calc(100vh - 50px);
  background-color: #232323;
  
  h1 {
    color: #fff;
    font-weight: normal;
    padding-top: 150px;
    text-align: center;
    margin: 0;
  }
  
  div {
    display: flex;
    justify-content: center;
    
    
    &:nth-last-child(2) {
      color: red;
      font-size:200px;
    }
    
    button {
      margin: 0;
      width: 200px;
    }
  }
`;
