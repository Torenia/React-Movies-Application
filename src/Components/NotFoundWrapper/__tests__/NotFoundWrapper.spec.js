import React from 'react';

import { NotFoundWrapper } from '../NotFoundWrapper';

describe('<NotFoundWrapper />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<NotFoundWrapper />);
  });

  it('should render a not found wrapper', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
