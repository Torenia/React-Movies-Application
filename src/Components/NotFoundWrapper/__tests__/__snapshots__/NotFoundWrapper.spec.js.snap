// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundWrapper /> should render a not found wrapper 1`] = `
.c0 {
  width: 100%;
  height: calc(100vh - 50px);
  background-color: #232323;
}

.c0 h1 {
  color: #fff;
  font-weight: normal;
  padding-top: 150px;
  text-align: center;
  margin: 0;
}

.c0 div {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c0 div:nth-last-child(2) {
  color: red;
  font-size: 200px;
}

.c0 div button {
  margin: 0;
  width: 200px;
}

<div
  className="c0"
/>
`;
